### PR TITLE
Admin rtl menu

### PR DESCRIPTION
**Pull request by deepaksinghgusain227**
#5537

Fixed New admin theme - RTL menu

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -22,6 +22,18 @@
...
+
+        $paddingNavbar = 'padding-left:56px';
+        $paddingNavbarOpen = 'padding-left:200px';
+
+        if(core()->getCurrentLocale() && core()->getCurrentLocale()->direction == 'rtl') {
+            $paddingNavbar = 'padding-right:56px';
+            $paddingNavbarOpen = 'padding-right:200px';
+        }
+
+        @endphp
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/layouts/master.blade.php:36]: This should be handled with scss. No, if cases for RTL.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/layouts/master.blade.php (hunk 1 above)
Comment (deepaksinghgusain227) [reply packages/Webkul/Admin/src/Resources/views/layouts/master.blade.php:36]: ok sir

**Comment by singhgagan03** [comment]
conflict not resolved.

**Comment by devansh-webkul** [comment]
Screenshot(2)

Getting white when navigating.

**Comment by singhgagan03** [comment]
working now

**Pull request merged**